src/utils/fetches.ts

```diff
@@ -20,12 +22,17 @@ export const fetchWithCredentials = (
   );
 };
 
+export function parseLargeNumbersAsStrings(value: any) {
+  return isSafeNumber(value, { approx: false })
+    ? parseFloat(value) // Smaller numbers are kept as Javascript numbers
+    : new LosslessNumber(value).toString(); // Large numbers are safely stringified
+}
+
```

Comment:
actually based on this example https://github.com/josdejong/lossless-json?tab=readme-ov-file#lossless-json couldn't we just call `parse()` and that's it?

Reply:
So it's because we _**want**_ the large numbers as a string (instead of a BigInt which would render in the UI as `1000000000000001n`). In their example they are using their own `stringify` to render it as a large number without a string, but we would need to refactor a lot of the codebase to use that instead of our existing JSON serialisation logic.

Reply:
Got caught out that stringify is actually their own function